modules/db/src/main/CursorExt.scala

```diff
@@ -20,7 +20,7 @@ trait CursorExt { self: dsl =>
     def list(): Fu[List[A]] = list(none)
 
     // like headOption, but with stopOnError defaulting to false
-    def uno: Fu[Option[A]] =
-      c.collect[Iterable](1, stopOnError = false).map(_.headOption)
+    def uno: Fu[Option[A]] = c.collect[Iterable](
```

Comment:
If I remember correctly, it's about malformed mongodb document. `.headOption` will throw an exception if the mongodb doc exists but the reader can't map it. `.uno` returns None in this case, which I found to be usually preferable.

For instance, a game page without a chat is better than no game page at all.